StantUiIosLibrary/Classes/Cells/ProfileWithImageCell.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ProfileWithImageCell.swift
+//  Pods-StantUiIosLibraryDemo
+//
+//  Created by Alexandre Henrique on 7/3/19.
+//
+
+import UIKit
+
+public class ProfileWithImageCell: UITableViewCell {
```

Comment:
Same comment about the name. = Intern Cell / Person.

Reply:
Done!